server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/service/UserService.java

```diff
@@ -88,8 +85,7 @@ public String upsertUser(String username, char[] password, Response response) th
 
     private void usernamePreconditions(String username) {
         try {
-            Preconditions.checkArgument(!Strings.isNullOrEmpty(username));
-            Preconditions.checkArgument(username.length() <= MAXIMUM_MAIL_ADDRESS_LENGTH);
+            org.apache.james.core.User.fromUsername(username);
```

Comment:
Import `User` class?

Reply:
Sadly, there is already another `User` class imported

Reply:
@rouazana work should fix this.